### ISSUE TITLE
Make npm & registry target more robust

### ISSUE THREAD

**Issue by HazAT** [getsentry/craft#166]
When publishing we should make the `npm` and `registry` target not fail if the version already exists.
It makes the publishing process annoying if it stops working somewhere middle in the process.

`npm`
```
npm: npm ERR! 403 403 Forbidden - PUT https://registry.npmjs.org/@sentry%2fangular - You cannot publish over the previously published versions: 6.0.0.
```

`registry`
```
ℹ info [registry] › Adding the version file to the registry for canonical name "npm:@sentry/browser"...
✖ error Error: Version file for "6.0.0" already exists. Aborting.
```

**Comment by BYK** [comment]
We are tracking this internally here: https://app.asana.com/0/1198192131329257/1198192131329297

@chadwhitacre we should probably port that issue here?

**Comment by BYK** [comment]
This is addressed more generically by #192 and #197 so closing.
